manage/groups/access-levels.rst

```diff
@@ -20,7 +20,7 @@ Within each group, the different access levels allow an agent to...
 :OVERVIEW: ...see ticket overviews (but not ticket details)
 :FULL:     ...all of the above **and be assigned tickets**
```

Comment:
How about

```suggestion
:FULL:     ...all of the above **and be assigned / receive notifications for tickets**
```

instead of the note below?

Reply:
Thanks for your input, will be address with commit 229e5a8c55fcf709e0eadb232d165c1959a47a63